### DISCUSSION TITLE
Not working sub-80pct? (M1 MBP)

### DISCUSSION THREAD

**Discussion by nickmv5** [mhaeuser/Battery-Toolkit#34] [General]
Hey there! I installed this utility earlier this week and so far it's been working great, until this morning when I tried to lower the percentage limit to 70. Previously I had the settings at 80/80, so I just shifted them to 70/70.

As I type this message, my M1 Macbook Pro is at 73% and continues to charge, which is opposite of expected behavior. Any ideas as to what's going on? Battery-Toolkit's status in the menubar shows "Charging on hold" while the Apple battery menu clearly shows it's charging and the remaining time until it's done charging. 

I quit and relaunched the utility just in case, same result.

Something's off.

**Comment by mhaeuser** [comment]
Is Optimized Charging disabled?

**Reply by nickmv5** [reply]
Sure enough I had missed that! Just disabled!

**Comment by mhaeuser** [comment]
OK, feel free to re-open if you continue running into issues.
